gaurun/log.go

```diff
@@ -45,7 +46,11 @@ type Reopener interface {
 	Reopen() error
 }
 
-func InitLog(outString, levelString string) (zap.Logger, Reopener, error) {
+func LocalTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
+	enc.AppendString(t.Format("2006/01/02 15:04:05 MST"))
+}
+
+func InitLog(outString, levelString string) (*zap.Logger, Reopener, error) {
```

Comment:
Why not pass the `writer reopen.Writer` ? Then you can reuse it for test by passing `reopen.Discard`